OpenRA.Mods.TS/UtilityCommands/LegacySequenceImporter.cs

```diff
@@ -0,0 +1,275 @@
+				if (!string.IsNullOrEmpty(prone))
+				{
+					Console.WriteLine("\tprone-stand:");
+					ConvertStartLengthFacings(prone);
+				}
+
+				var crawl = sequenceSection.GetValue("Crawl", string.Empty);
+				if (!string.IsNullOrEmpty(crawl))
+				{
+					Console.WriteLine("\tprone-run:");
+					ConvertStartLengthFacings(crawl);
+				}
+
+				var fireProne = sequenceSection.GetValue("FireProne", string.Empty);
+				if (!string.IsNullOrEmpty(fireProne))
+				{
+					Console.WriteLine("\tprone-shoot:");
+					ConvertStartLengthFacings(fireProne);
+				}
+
+				var fireUp = sequenceSection.GetValue("FireUp", string.Empty);
+				if (!string.IsNullOrEmpty(fireUp))
+				{
+					Console.WriteLine("\tshoot:");
+					ConvertStartLengthFacings(fireUp);
+				}
+
+				var idle1 = sequenceSection.GetValue("Idle1", string.Empty);
+				if (!string.IsNullOrEmpty(idle1))
+				{
+					Console.WriteLine("\tidle1:");
+					ConvertStartLengthFacings(idle1);
+				}
+
+				var idle2 = sequenceSection.GetValue("Idle2", string.Empty);
+				if (!string.IsNullOrEmpty(idle2))
+				{
+					Console.WriteLine("\tidle2:");
+					ConvertStartLengthFacings(idle2);
+				}
+
+				var die1 = sequenceSection.GetValue("Die1", string.Empty);
+				if (!string.IsNullOrEmpty(die1))
+				{
+					Console.WriteLine("\tdie1:");
+					ConvertStartLengthFacings(die1);
+				}
+
+				var die2 = sequenceSection.GetValue("Die2", string.Empty);
+				if (!string.IsNullOrEmpty(die2))
+				{
+					Console.WriteLine("\tdie2:");
+					ConvertStartLengthFacings(die2);
+				}
+
+				var die3 = sequenceSection.GetValue("Die3", string.Empty);
+				if (!string.IsNullOrEmpty(die3))
+				{
+					Console.WriteLine("\tdie3:");
+					ConvertStartLengthFacings(die3);
+				}
+
+				var die4 = sequenceSection.GetValue("Die4", string.Empty);
+				if (!string.IsNullOrEmpty(die4))
+				{
+					Console.WriteLine("\tdie4:");
+					ConvertStartLengthFacings(die4);
+				}
+
+				var die5 = sequenceSection.GetValue("Die5", string.Empty);
+				if (!string.IsNullOrEmpty(die5))
+				{
+					Console.WriteLine("\tdie5:");
+					ConvertStartLengthFacings(die5);
+				}
+
+				var cheer = sequenceSection.GetValue("Cheer", string.Empty);
+				if (!string.IsNullOrEmpty(cheer))
+				{
+					Console.WriteLine("\tcheer:");
+					ConvertStartLengthFacings(cheer);
+				}
+
+				var panic = sequenceSection.GetValue("Panic", string.Empty);
+				if (!string.IsNullOrEmpty(panic))
+				{
+					Console.WriteLine("\tpanic-stand:");
+					ConvertStartLengthFacings(panic);
+					Console.WriteLine("\tpanic-run:");
+					ConvertStartLengthFacings(panic);
+				}
+			}
+
+			Console.WriteLine();
+		}
+
+		void ConvertStartLengthFacings(string input)
+		{
+			var splitting = input.Split(',');
+			if (splitting.Length >= 3)
+			{
+				Console.WriteLine("\t\tStart: " + splitting[0]);
+				Console.WriteLine("\t\tLength: " + splitting[1]);
+
+				// TODO splitting[2] - splitting[1] may be Stride:
```

Comment:
Oh, apparently there are cases in RA2 where this is used (crawl's first frames are reused as prone in RA2). https://github.com/Phrohdoh/oramod-ra2/blob/master/extra-files/art.ini#L9637